code/game/machinery/Freezer.dm

```diff
@@ -253,12 +254,14 @@
 		if (node)
 			node.initialize()
 			node.build_network()
-		verbs += rotate_verbs
 	else
-		node.disconnect(src)
-		qdel(network)
-		network = null
-		node = null
+		verbs += rotate_verbs
```

Comment:
Should this line have moved beneath the `else`?

Reply:
yeah

Reply:
Oh, I misread the conditions the first time through, neat